cmd/node/main.go

```diff
@@ -353,20 +353,6 @@ VERSION:
 			"and will have a full history over epochs.",
 	}
 
-	numEpochsToSave = cli.Uint64Flag{
```

Comment:
why did you remove the flags?

Reply:
no longer required, they are defined in the config.toml and observed that we never used them as flags. Initially I thought we are mistakenly changed those when running the node. Didn't turned out that we specify them in the testnet scripts.

Reply:
Ok, just wondering if one would have used these flags in a custom node script. It is easier to configure these settings via flags.

Reply:
It is very easy to mess things out. I was even wondering if we can remove some configs values from config.toml (regarding this area) as to further decrease the chances of breaking things up.